artemis/modules/joomla_scanner.py

```diff
@@ -40,7 +40,8 @@ def run(self, current_task: Task) -> None:
             joomla_version = match.group(1)
             result["joomla_version"] = joomla_version
             # Get latest release in repo from GitHub API
-            gh_api_response = requests.get("https://api.github.com/repos/joomla/joomla-cms/releases/latest")
+            gh_api_response = FallbackAPICache.URL_JOOMLA_LATEST_RELEASE.get()
```

Comment:
inb4 - I see that you wanted cls fields solely for this reason, but still, you might just use magic methods when having dict